public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -630,7 +633,7 @@ figure.image-display
 +makeExample('toh-5/ts/styles.1.css', '', 'styles.css (App Styles)')(format=".")
 
 :marked
-  Reference this stylesheet within the `index.html` in the traditional manner.
+  Reference this stylesheet within the `index.html` (if not there yet) in the traditional manner.
```

Comment:
I know what you mean (because it is there in the QuickStart distribution). But I think it's an awkward parenthetical and not necessary. Please retract.